tendrl/gluster_integration/message/callback.py

```diff
@@ -406,8 +407,15 @@ def peer_connect(self, event):
         )
 
     def peer_disconnect(self, event):
+        fetched_peers = NS.gluster.objects.Peer().load_all()
```

Comment:
I had a chat with Atin today morning and looks like we should not be removing the graphite data and mark peer deleted from monitoring integration in case of peer disconnect. This is a temporary event and peer might connect back soon.

PERR_DETACH is the event may be, to do these thing but at the moment this event is not handled in tendrl. I would suggest to remove the logic of removing graphite data and marking the peer as deleted on this event.

@atinmu can you please comment and confirm the same.
@Tendrl/tendrl-core need comments on this to remove the marking of peer as deleted and call of `delete_resource_from_graphite`.

Reply:
peer delete and peer disconnect are two different events. Disconnection can happen because of (a) if the node is down or (b) glusterd instance running on the node is down or (c) there's a n/w disconnect . In any of these cases, glusterd doesn't wipe off the peerinfo object from its peer list. On further connect event the peer status is set back to connect. So marking the peer deleted in case of a disconnect event doesn't sound correct to me.